src/logger.c

```diff
@@ -31,6 +35,11 @@ void logger_init(log_level_t log_level)
 		.stopbits = US_MR_NBSTOP_1_BIT
 	};
 	serial_mdw_init_interface(LOGGER_SERIAL_INTERFACE, &serial_option, TIMESTAMP_USED);
+
+	// Initialize UDP client
+	IP4_ADDR(&ipaddr, RPI_IP_ADDR_A, RPI_IP_ADDR_B, RPI_IP_ADDR_C, RPI_IP_ADDR_D);
```

Comment:
ip address of destination should be a parameter and not fixed

Reply:
Same as for the destination port